assets/css/style.css

```diff
@@ -524,3 +694,208 @@ svg {
+    padding: 0;
+    display: flex;
+    justify-content: center;
+  }
+
+  .hero .hero-content {
+    width: 75%;
+    max-width: 1100px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+  .hero .social-links,
+  .about-me .social-links {
+    margin: 0;
+  }
+
+  .my-works {
+    padding: 142px 24px;
+    grid-gap: 142px;
+  }
+
+  .my-works .work-item {
+    padding: 24px;
+    flex-direction: row;
+    align-items: flex-start;
+    border-radius: 24px;
+    transition: all 0.3s ease;
+  }
+
+  .my-works .work-item:hover {
+    cursor: pointer;
+    border: 1px solid #a7aeff;
+    filter: drop-shadow(0 48px 48px rgba(37, 47, 137, 0.08));
+  }
+
+  .my-works .work-item:hover .view-work-btn:enabled,
+  .my-works .work-item:hover .view-work-btn:hover {
+    background-color: #4053fc;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+    color: #fff;
+  }
+
+  .my-works .work-item.reverse-item {
+    flex-direction: row-reverse;
+  }
+
+  .my-works .work-item .work-item-content {
+    margin-left: 24px;
+    flex-grow: 1;
+    padding: 22px;
+  }
+
+  .my-works .work-item.reverse-item .work-item-content {
+    margin-left: 0;
+    margin-right: 24px;
+  }
+
+  .my-works .work-item img {
+    flex-grow: 1;
+    width: 50%;
+  }
+
+  .my-works .work-item .work-item-content .work-title {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .my-works .work-item .work-item-content .work-subtitle {
+    font-size: 18px;
+    line-height: 24px;
+    font-weight: 500;
+  }
+
+  .my-works .work-item .work-details {
+    margin-bottom: 28px;
+  }
+
+  .my-works .work-item .work-technologies {
+    margin-bottom: 16px;
+  }
+
+  .about-me-container {
+    flex-direction: row;
+  }
+
+  .about-me-container .top {
+    flex-grow: 1;
+    padding-right: 48px;
+  }
+
+  .about-me-container .bottom {
+    flex-grow: 1.5;
+  }
+
+  .about-me-container .top .title {
+    width: 120px;
+  }
+
+  .about-me-container .top .introduction {
+    margin-bottom: 12px;
+  }
+
+  .about-me-container .top .sub-title {
+    margin-bottom: 24px;
+  }
+
+  .about-me-container .top .resume-btn {
+    margin: 32px 0;
+  }
+
+  .about-me-container .bottom .knowledge-area.active {
+    margin-bottom: 12px;
+  }
+
+  .about-me-container .bottom .technical-knowledge-list {
+    display: flex;
+    flex: 1;
+  }
+
+  .footer .contact-form li {
+    justify-content: center;
+  }
+
+  .about-me-container .bottom .technical-knowledge-list li {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    min-height: 120px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-right: 12px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    flex-basis: 17.5%;
+  }
+
+  .about-me-container .bottom .technical-knowledge-list li img {
+    padding-bottom: 16px;
+  }
+
+  .footer .footer-content {
+    max-width: 600px;
+    margin: auto;
+  }
+
+  .footer .contact-form {
+    padding: 0 24px;
+  }
+}

```

Comment:
- [ ] [REQUIRED]You have done a great job concerning the desktop version, however, you need to stick to the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as much as possible.
The landing page looks different,
The card/work sections need to be centered. You can use margin-left and right: auto or give it a specific margin on the left and right.
Your contact section also looks different from the one provided in the template.
Kindly ensure that your entire design looks more like the figma design provided.

Take a look at these images:

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![3](https://user-images.githubusercontent.com/58692590/149608524-c79cb6f5-5175-4315-95ab-7d62ce17a935.png) | ![33](https://user-images.githubusercontent.com/58692590/149608537-4c6f1756-2984-48d0-8737-73d7ebb90d61.png)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![5](https://user-images.githubusercontent.com/58692590/149608770-022015a3-b3e2-40bc-8f0e-2ef0ce529921.png) | ![44](https://user-images.githubusercontent.com/58692590/149608795-7890b87a-f58f-4729-8968-c21f0f2a6a1f.png)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![7](https://user-images.githubusercontent.com/58692590/149608839-d0a01c80-77cb-434a-82a1-1c5beec83e4c.png) | ![55](https://user-images.githubusercontent.com/58692590/149608851-4a74bb8b-3da1-49e2-a3f1-a14a6e4b8e25.png)